src/components/presentation-summary-form.js

```diff
@@ -274,35 +274,39 @@ const PresentationSummaryForm = (props) => {
                         />
                     </div>
                 </div>
-                <div className="row form-group">
-                    <div className="col-md-12">
-                        <label> {T.translate("edit_presentation.format")} </label>
-                        <p> {event_types_limits} </p>
-                        <Dropdown
-                            id="type_id"
-                            value={entity.type_id}
-                            onChange={handleChange}
-                            placeholder={T.translate("general.placeholders.select_one")}
-                            options={event_types_ddl}
-                            disabled={!!entity.id}
-                            error={hasErrors('type_id')}
-                        />
+                {selectionPlanSettings?.CFP_PRESENTATION_SUMMARY_HIDE_TRACK_SELECTION !== '1' &&
```

Comment:
condtion is 
selectionPlanSettings?.CFP_PRESENTATION_SUMMARY_HIDE_TRACK_SELECTION !== '1' && event_types_ddl.length > 1